18 Normalizr/src/model/mappers/countryMapper.ts

```diff
@@ -0,0 +1,19 @@
+import { Country } from '../api/country';
+import { CountryView } from '../view/countryView';
+
+class CountryMapper {
+  mapCountryToCountryView(country: Country): CountryView {
+    return {
+      id: country.id,
+      name: country.name
+    }
```

Comment:
You are right, we can return original object and in the future, map additional props or some format